### ISSUE TITLE
error on change skin

### ISSUE THREAD

**Issue by KonoromiHimaries** [games647/ChangeSkin#162]
[//]: # (Lines in this format are considered as comments and will not be displayed.)
[//]: #
[//]: # (Before reporting an issue make sure you are running the latest build and checked for duplicate issues!)

### Steps/models to reproduce:
[//]: # (The actions that cause the issue. Please explain it in detail)
install https://yggdrasil-network.github.io/about.html
install https://github.com/Akarin-project/Akarin
try join to server on yggdrasil ipv6
use /skin random_nick

### Environment description
[//]: # (Standalone server/Bungeecord network with version and build number, SQLite/MySQL, ...)
https://timings.aikar.co/?id=286d214fb1d446419331e5b7fc498e16

### Error Log:
[Hastebin](https://hastebin.com/) / [Gist](https://gist.github.com/) link of the error (if any)

```
[13:00:35 WARN]: [ChangeSkin] Task #281 for ChangeSkin v3.0 generated an exception
com.comphenix.protocol.reflect.FieldAccessException: No field with type int exists in class PacketPlayOutRespawn.
        at com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:363) ~[?:?]
        at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:343) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendPacketsSelf(SkinUpdater.java:169) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendUpdateSelf(SkinUpdater.java:121) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendUpdate(SkinUpdater.java:102) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.onInstantUpdate(SkinUpdater.java:94) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.run(SkinUpdater.java:80) ~[?:?]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:84) ~[akarin-1.13.2.jar:git-Akarin-"18cd8f6"]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:449) ~[akarin-1.13.2.jar:git-Akarin-"18cd8f6"]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1015) ~[akarin-1.13.2.jar:git-Akarin-"18cd8f6"]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:440) ~[akarin-1.13.2.jar:git-Akarin-"18cd8f6"]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:945) ~[akarin-1.13.2.jar:git-Akarin-"18cd8f6"]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:842) ~[akarin-1.13.2.jar:git-Akarin-"18cd8f6"]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_201]
```

### Configuration:
[//]: # (Remember to delete any sensitive data)
[Hastebin](https://hastebin.com/) / [Gist](https://gist.github.com/) link of your config.yml file 

default config

**Comment by Martin-H2** [comment]
confirmed on Spigot 1.13.2 / Java 12 openJDK / windows 64bit

`>WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.comphenix.net.sf.cglib.core.ReflectUtils$1 (file:/D:/games/MineCraft_Server/build_server_1.13.2/plugins/ProtocolLib.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byt
e[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.comphenix.net.sf.cglib.core.ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[00:01:21 INFO]: [AutoWhitelist] Executing scheduled whitelist reload.
[00:01:21 INFO]: [AutoWhitelist] Whitelist is ON
[00:01:21 INFO]: [AutoWhitelist] Configuration reloaded.
[00:01:21 INFO]: * Rei joined the game.
[00:02:37 WARN]: [ChangeSkin] Task #249 for ChangeSkin v3.0 generated an exception
com.comphenix.protocol.reflect.FieldAccessException: No field with type int exists in class PacketPlayOutRespawn.
        at com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:363) ~[?:?]
        at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:343) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendPacketsSelf(SkinUpdater.java:169) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendUpdateSelf(SkinUpdater.java:121) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendUpdate(SkinUpdater.java:102) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.onInstantUpdate(SkinUpdater.java:94) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.run(SkinUpdater.java:80) ~[?:?]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.13.2.jar:git-Spigot-f09662d-51d62de]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:392) [spigot-1.13.2.jar:git-Spigot-f09662d-51d62de]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:889) [spigot-1.13.2.jar:git-Spigot-f09662d-51d62de]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar:git-Spigot-f09662d-51d62de]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot-1.13.2.jar:git-Spigot-f09662d-51d62de]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot-1.13.2.jar:git-Spigot-f09662d-51d62de]
        at java.lang.Thread.run(Thread.java:835) [?:?]
>

`

**Comment by mirolm** [comment]
Happens also in Spigot-1eece4f-84400e6 (MC: 1.14), latest ProtocolLib v4.5.0-SNAPSHOT from gradle branch and ChangeSkin v3.1-SNAPSHOT-bece00f.

``[22:56:58] [Server thread/WARN]: [ChangeSkin] Task #27 for ChangeSkin v3.1-SNAPSHOT-bece00f generated an exception
com.comphenix.protocol.reflect.FieldAccessException: No field with type net.minecraft.server.v1_14_R1.EnumDifficulty exists in class PacketPlayOutRespawn.
	at com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:363) ~[?:?]
	at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:343) ~[?:?]
	at com.github.games647.changeskin.bukkit.task.SkinApplier.createRespawnPacket(SkinApplier.java:222) ~[?:?]
	at com.github.games647.changeskin.bukkit.task.SkinApplier.sendPacketsSelf(SkinApplier.java:164) ~[?:?]
	at com.github.games647.changeskin.bukkit.task.SkinApplier.sendUpdateSelf(SkinApplier.java:116) ~[?:?]
	at com.github.games647.changeskin.bukkit.task.SkinApplier.applyInstantUpdate(SkinApplier.java:86) ~[?:?]
	at com.github.games647.changeskin.core.shared.task.SharedApplier.applySkin(SharedApplier.java:27) ~[?:?]
	at com.github.games647.changeskin.bukkit.task.SkinApplier.run(SkinApplier.java:74) ~[?:?]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.14.jar:git-Spigot-1eece4f-84400e6]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:394) [spigot-1.14.jar:git-Spigot-1eece4f-84400e6]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1009) [spigot-1.14.jar:git-Spigot-1eece4f-84400e6]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:396) [spigot-1.14.jar:git-Spigot-1eece4f-84400e6]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:957) [spigot-1.14.jar:git-Spigot-1eece4f-84400e6]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:802) [spigot-1.14.jar:git-Spigot-1eece4f-84400e6]
	at java.lang.Thread.run(Thread.java:835) [?:?]
``
I think skin changes, but only the exception is generated.

**Comment by CommanderTvis** [comment]
Confirmed on Ubuntu (Liberica JDK 8u212)
```
[09:41:04 WARN]: [ChangeSkin] Task #48 for ChangeSkin v3.0 generated an exception
com.comphenix.protocol.reflect.FieldAccessException: No field with type int exists in class PacketPlayOutRespawn.
        at com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:363) ~[?:?]
        at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:343) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendPacketsSelf(SkinUpdater.java:169) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendUpdateSelf(SkinUpdater.java:121) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.sendUpdate(SkinUpdater.java:102) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.onInstantUpdate(SkinUpdater.java:94) ~[?:?]
        at com.github.games647.changeskin.bukkit.tasks.SkinUpdater.run(SkinUpdater.java:80) ~[?:?]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:81) ~[core.jar:git-Spigot-1a3504a-dfa7583]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:392) [core.jar:git-Spigot-1a3504a-dfa7583]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:889) [core.jar:git-Spigot-1a3504a-dfa7583]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [core.jar:git-Spigot-1a3504a-dfa7583]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [core.jar:git-Spigot-1a3504a-dfa7583]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [core.jar:git-Spigot-1a3504a-dfa7583]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
```

**Comment by games647** [comment]
@KonoromiHimaries 
@Martin-H2 
@CommanderTvis 

Duplicate of #154 - fixed a long time ago. 

@mirolm New bug that occurred in 1.14. Fixed in e48d95895f6bf0281571f9557050fa4fc423a70f. You can look at the development builds for a fixed version.
